examples/rhscl_maria/config.yaml

```diff
@@ -0,0 +1,18 @@
+document: modularity-testing
+version: 1
+name: mariadb
+modulemd-url: https://src.fedoraproject.org/modules/base-runtime/raw/master/f/base-runtime.yaml
```

Comment:
Hi @TomasTomecek it is more less just stub, because I wanted to show to hhorak, how to use MTF for testing collections. And now I expect that there will be some modulemd file (although empty) so that I've choosed some generic, and baseruntime is the most minimal one. This example allows to test collection without any changes of MTF